docs/getting-started/getting-started-with-docker.rst

```diff
@@ -22,15 +24,143 @@ There is a good chance that you have heard of containerization. A container is e
 
 Docker is a well-known type of containerization software. It provides a fast environment for both users and developers. A developer can package software and its dependencies into an Docker image, and then share that Docker image with users. Users who download your Docker image and run it with the Docker program will be able to run the software you packaged without having to handle the installation of your program's prerequisites or anything else; it's ready to go as-is.
 
-Docker images are usually shared on registries such as Quay.io, Docker Hub, and GitLab. When a user downloads an image and runs it locally, the Docker Engine spins up an instance of the Docker image. This instance is called a Docker container. You can think of an image as a template, and a container as something made from that template.
+When a user runs a Docker image locally, the Docker Engine spins up an instance of the Docker image. This instance is called a Docker container. You can think of an image as a template, and a container as something made from that template. Note that the filesystem of a Docker container is sequestered from the rest of your computer, so you will either want the container to already have any extra files you might need, or get those files into the Docker container by "mounting" them. We will briefly talking about how to mount files in this tutorial, but keep in mind that workflow languages handle this for us automatically, as will be shown in :doc:`Getting Started with Nextflow <getting-started-with-nextflow>`, :doc:`Getting Started with CWL <getting-started-with-cwl>`, and :doc:`Getting Started with WDL <getting-started-with-wdl>`. But we don't always need to mount files: when we talk about how to create a Dockerfile, you will learn how to create a Docker image that already has the files you need -- whether it's a simple 3 line Python script saved as a text file, or a complicated compiled program with lots of prerequisites.
```

Comment:
Not sure about the word "sequestered". Is that standard Docker terminology?

Reply:
AFAIK it is not official terminology but it is common terminology, at least when people are describing the filesystem, such as [this blog post](https://www.fermyon.com/blog/webassembly-vs-containers):
> Were we to execute this application, the container runtime would unpack each of these layers in sequence – the Linux distro, the supporting files, and then our program. Using core technologies of the Linux kernel, the resources are all unpacked into a sequestered filesystem that is isolated from the host’s filesystem, and then the kernel itself would execute our program as if this sequestered filesystem was the root filesystem of the program.

I'm also not sure how to describe it without using that word. "Separate" is not entirely correct.